Blink Arduino sample fails to compile on ESP32S3

Sample used without any changes:
https://github.com/FastLED/FastLED/tree/master/examples/Blink

My custom sample which was working earlier, is not compiling anymore. So tested with the Blink sample and same issues.
You don't need hardware to test it. Set the MCU to S3 device and try compiling.

Compiler Error output. 
```
Compiling libraries...
In file included from c:\Users\sukes\Documents\Arduino\libraries\FastLED\src/FastLED.h:75,
                 from C:\Users\sukes\AppData\Local\Temp\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\Blink\Blink.ino:5:
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src/fastspi.h:157:23: note: '#pragma message: No hardware SPI pins defined.  All SPI access will default to bitbanged output'
  157 | #      pragma message "No hardware SPI pins defined.  All SPI access will default to bitbanged output"
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Compiling library "FastLED"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\colorutils.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\colorutils.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\bitswap.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\bitswap.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\FastLED.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\FastLED.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\colorpalettes.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\colorpalettes.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\hsv2rgb.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\hsv2rgb.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\lib8tion.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\lib8tion.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\noise.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\noise.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\platforms\\esp\\32\\clockless_rmt_esp32.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\platforms\\esp\\32\\clockless_rmt_esp32.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\platforms.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\platforms.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\power_mgt.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\power_mgt.cpp.o"
"C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\xtensa-esp32s3-elf-gcc\\esp-12.2.0_20230208/bin/xtensa-esp32s3-elf-g++" -MMD -c "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/cpp_flags" -w -Os -DF_CPU=240000000L -DARDUINO=10607 -DARDUINO_TINYS3 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"TINYS3\"" "-DARDUINO_VARIANT=\"um_tinys3\"" -DARDUINO_PARTITION_default_8MB "-DARDUINO_HOST_OS=\"windows\"" "-DARDUINO_FQBN=\"esp32:esp32:tinys3:UploadSpeed=921600,USBMode=hwcdc,CDCOnBoot=cdc,MSCOnBoot=default,DFUOnBoot=default,UploadMode=default,CPUFreq=240,FlashMode=qio,PartitionScheme=default_8MB,DebugLevel=none,PSRAM=enabled,LoopCore=1,EventsCore=1,EraseFlash=none\"" -DESP32 -DCORE_DEBUG_LEVEL=0 -DARDUINO_RUNNING_CORE=1 -DARDUINO_EVENT_RUNNING_CORE=1 -DBOARD_HAS_PSRAM -DARDUINO_USB_MODE=1 -DARDUINO_USB_CDC_ON_BOOT=1 -DARDUINO_USB_MSC_ON_BOOT=0 -DARDUINO_USB_DFU_ON_BOOT=0 "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/defines" "-IC:\\Users\\sukes\\AppData\\Local\\Temp\\.arduinoIDE-unsaved2023921-15596-mlo18f.socmm\\Blink" -iprefix "C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/include/" "@C:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/flags/includes" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\tools\\esp32-arduino-libs\\idf-release_v5.1-6b1f40b9bf/esp32s3/qio_qspi/include" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\cores\\esp32" "-IC:\\Users\\sukes\\AppData\\Local\\Arduino15\\packages\\esp32\\hardware\\esp32\\3.0.0-alpha2\\variants\\um_tinys3" "-Ic:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src" "@C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7/build_opt.h" "c:\\Users\\sukes\\Documents\\Arduino\\libraries\\FastLED\\src\\wiring.cpp" -o "C:\\Users\\sukes\\AppData\\Local\\Temp\\arduino\\sketches\\0BC9D72F78793541A3C87EDF8B9A09E7\\libraries\\FastLED\\wiring.cpp.o"
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp: In function 'void fill_raw_2dnoise8(uint8_t*, int, int, uint8_t, q44, fract8, int, uint16_t, int, uint16_t, int, uint16_t)':
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp:623:100: error: ambiguous overload for 'operator*' (operand types are 'q44' {aka 'q<unsigned char, 4, 4>'} and 'int')
  623 |     fill_raw_2dnoise8(pData, width, height, octaves-1, freq44, amplitude, skip+1, x*freq44, freq44 * scalex, y*freq44, freq44 * scaley, time);
      |                                                                                             ~~~~~~ ^ ~~~~~~
      |                                                                                             |        |
      |                                                                                             |        int
      |                                                                                             q44 {aka q<unsigned char, 4, 4>}
In file included from c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\pixeltypes.h:7,
                 from c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\controller.h:9,
                 from c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\FastLED.h:57,
                 from c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp:6:
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:909:14: note: candidate: 'uint32_t q<T, F, I>::operator*(uint32_t) [with T = unsigned char; int F = 4; int I = 4; uint32_t = long unsigned int]'
  909 |     uint32_t operator*(uint32_t v) { return (v*i) + ((v*f)>>F); }
      |              ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:911:14: note: candidate: 'uint16_t q<T, F, I>::operator*(uint16_t) [with T = unsigned char; int F = 4; int I = 4; uint16_t = short unsigned int]'
  911 |     uint16_t operator*(uint16_t v) { return (v*i) + ((v*f)>>F); }
      |              ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:913:13: note: candidate: 'int32_t q<T, F, I>::operator*(int32_t) [with T = unsigned char; int F = 4; int I = 4; int32_t = long int]'
  913 |     int32_t operator*(int32_t v) { return (v*i) + ((v*f)>>F); }
      |             ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:915:13: note: candidate: 'int16_t q<T, F, I>::operator*(int16_t) [with T = unsigned char; int F = 4; int I = 4; int16_t = short int]'
  915 |     int16_t operator*(int16_t v) { return (v*i) + ((v*f)>>F); }
      |             ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp:623:127: error: ambiguous overload for 'operator*' (operand types are 'q44' {aka 'q<unsigned char, 4, 4>'} and 'int')
  623 |     fill_raw_2dnoise8(pData, width, height, octaves-1, freq44, amplitude, skip+1, x*freq44, freq44 * scalex, y*freq44, freq44 * scaley, time);
      |                                                                                                                        ~~~~~~ ^ ~~~~~~
      |                                                                                                                        |        |
      |                                                                                                                        |        int
      |                                                                                                                        q44 {aka q<unsigned char, 4, 4>}
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:909:14: note: candidate: 'uint32_t q<T, F, I>::operator*(uint32_t) [with T = unsigned char; int F = 4; int I = 4; uint32_t = long unsigned int]'
  909 |     uint32_t operator*(uint32_t v) { return (v*i) + ((v*f)>>F); }
      |              ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:911:14: note: candidate: 'uint16_t q<T, F, I>::operator*(uint16_t) [with T = unsigned char; int F = 4; int I = 4; uint16_t = short unsigned int]'
  911 |     uint16_t operator*(uint16_t v) { return (v*i) + ((v*f)>>F); }
      |              ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:913:13: note: candidate: 'int32_t q<T, F, I>::operator*(int32_t) [with T = unsigned char; int F = 4; int I = 4; int32_t = long int]'
  913 |     int32_t operator*(int32_t v) { return (v*i) + ((v*f)>>F); }
      |             ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:915:13: note: candidate: 'int16_t q<T, F, I>::operator*(int16_t) [with T = unsigned char; int F = 4; int I = 4; int16_t = short int]'
  915 |     int16_t operator*(int16_t v) { return (v*i) + ((v*f)>>F); }
      |             ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp: In function 'void fill_raw_2dnoise16(uint16_t*, int, int, uint8_t, q88, fract16, int, uint32_t, int, uint32_t, int, uint32_t)':
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp:661:101: error: ambiguous overload for 'operator*' (operand types are 'int' and 'q88' {aka 'q<short unsigned int, 8, 8>'})
  661 |     fill_raw_2dnoise16(pData, width, height, octaves-1, freq88, amplitude, skip, x *freq88 , scalex *freq88, y * freq88, scaley * freq88, time);
      |                                                                                              ~~~~~~ ^~~~~~~
      |                                                                                              |       |
      |                                                                                              int     q88 {aka q<short unsigned int, 8, 8>}
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:922:49: note: candidate: 'uint32_t operator*(uint32_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; uint32_t = long unsigned int]'
  922 | template<class T, int F, int I> static uint32_t operator*(uint32_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:923:49: note: candidate: 'uint16_t operator*(uint16_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; uint16_t = short unsigned int]'
  923 | template<class T, int F, int I> static uint16_t operator*(uint16_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:924:48: note: candidate: 'int32_t operator*(int32_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; int32_t = long int]'
  924 | template<class T, int F, int I> static int32_t operator*(int32_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:925:48: note: candidate: 'int16_t operator*(int16_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; int16_t = short int]'
  925 | template<class T, int F, int I> static int16_t operator*(int16_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp:661:129: error: ambiguous overload for 'operator*' (operand types are 'int' and 'q88' {aka 'q<short unsigned int, 8, 8>'})
  661 |     fill_raw_2dnoise16(pData, width, height, octaves-1, freq88, amplitude, skip, x *freq88 , scalex *freq88, y * freq88, scaley * freq88, time);
      |                                                                                                                          ~~~~~~ ^ ~~~~~~
      |                                                                                                                          |        |
      |                                                                                                                          int      q88 {aka q<short unsigned int, 8, 8>}
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:922:49: note: candidate: 'uint32_t operator*(uint32_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; uint32_t = long unsigned int]'
  922 | template<class T, int F, int I> static uint32_t operator*(uint32_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:923:49: note: candidate: 'uint16_t operator*(uint16_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; uint16_t = short unsigned int]'
  923 | template<class T, int F, int I> static uint16_t operator*(uint16_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:924:48: note: candidate: 'int32_t operator*(int32_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; int32_t = long int]'
  924 | template<class T, int F, int I> static int32_t operator*(int32_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:925:48: note: candidate: 'int16_t operator*(int16_t, q<T, F, I>&) [with T = short unsigned int; int F = 8; int I = 8; int16_t = short int]'
  925 | template<class T, int F, int I> static int16_t operator*(int16_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp: In function 'void fill_raw_2dnoise16into8(uint8_t*, int, int, uint8_t, q44, fract8, int, uint32_t, int, uint32_t, int, uint32_t)':
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp:699:106: error: ambiguous overload for 'operator*' (operand types are 'int' and 'q44' {aka 'q<unsigned char, 4, 4>'})
  699 |     fill_raw_2dnoise16into8(pData, width, height, octaves-1, freq44, amplitude, skip+1, x*freq44, scalex *freq44, y*freq44, scaley * freq44, time);
      |                                                                                                   ~~~~~~ ^~~~~~~
      |                                                                                                   |       |
      |                                                                                                   int     q44 {aka q<unsigned char, 4, 4>}
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:922:49: note: candidate: 'uint32_t operator*(uint32_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; uint32_t = long unsigned int]'
  922 | template<class T, int F, int I> static uint32_t operator*(uint32_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:923:49: note: candidate: 'uint16_t operator*(uint16_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; uint16_t = short unsigned int]'
  923 | template<class T, int F, int I> static uint16_t operator*(uint16_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:924:48: note: candidate: 'int32_t operator*(int32_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; int32_t = long int]'
  924 | template<class T, int F, int I> static int32_t operator*(int32_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:925:48: note: candidate: 'int16_t operator*(int16_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; int16_t = short int]'
  925 | template<class T, int F, int I> static int16_t operator*(int16_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\noise.cpp:699:132: error: ambiguous overload for 'operator*' (operand types are 'int' and 'q44' {aka 'q<unsigned char, 4, 4>'})
  699 |     fill_raw_2dnoise16into8(pData, width, height, octaves-1, freq44, amplitude, skip+1, x*freq44, scalex *freq44, y*freq44, scaley * freq44, time);
      |                                                                                                                             ~~~~~~ ^ ~~~~~~
      |                                                                                                                             |        |
      |                                                                                                                             int      q44 {aka q<unsigned char, 4, 4>}
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:922:49: note: candidate: 'uint32_t operator*(uint32_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; uint32_t = long unsigned int]'
  922 | template<class T, int F, int I> static uint32_t operator*(uint32_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:923:49: note: candidate: 'uint16_t operator*(uint16_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; uint16_t = short unsigned int]'
  923 | template<class T, int F, int I> static uint16_t operator*(uint16_t v, q<T,F,I> & q) { return q * v; }
      |                                                 ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:924:48: note: candidate: 'int32_t operator*(int32_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; int32_t = long int]'
  924 | template<class T, int F, int I> static int32_t operator*(int32_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\lib8tion.h:925:48: note: candidate: 'int16_t operator*(int16_t, q<T, F, I>&) [with T = unsigned char; int F = 4; int I = 4; int16_t = short int]'
  925 | template<class T, int F, int I> static int16_t operator*(int16_t v, q<T,F,I> & q) { return q * v; }
      |                                                ^~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp: In member function 'void ESP32RMTController::startOnChannel(int)':
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:266:29: error: 'RMTMEM' was not declared in this scope
  266 |         mRMT_mem_start = & (RMTMEM.chan[mRMT_channel].data32[0].val);
      |                             ^~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp: In member function 'void ESP32RMTController::tx_start()':
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:307:32: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'mem_rd_rst_n'
  307 |     RMT.chnconf0[mRMT_channel].mem_rd_rst_n = 1;
      |                                ^~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:308:32: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'mem_rd_rst_n'
  308 |     RMT.chnconf0[mRMT_channel].mem_rd_rst_n = 0;
      |                                ^~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:309:32: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'apb_mem_rst_n'
  309 |     RMT.chnconf0[mRMT_channel].apb_mem_rst_n = 1;
      |                                ^~~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:310:32: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'apb_mem_rst_n'
  310 |     RMT.chnconf0[mRMT_channel].apb_mem_rst_n = 0;
      |                                ^~~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:316:32: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'conf_update_n'
  316 |     RMT.chnconf0[mRMT_channel].conf_update_n = 1;
      |                                ^~~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:317:32: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'tx_start_n'
  317 |     RMT.chnconf0[mRMT_channel].tx_start_n = 1;
      |                                ^~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp: In static member function 'static void ESP32RMTController::doneOnChannel(rmt_channel_t, void*)':
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:347:5: error: 'gpio_matrix_out' was not declared in this scope; did you mean 'gpio_iomux_out'?
  347 |     gpio_matrix_out(pController->mPin, 0x100, 0, 0);
      |     ^~~~~~~~~~~~~~~
      |     gpio_iomux_out
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:368:27: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'tx_stop_n'
  368 |     RMT.chnconf0[channel].tx_stop_n = 1;
      |                           ^~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:369:27: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'conf_update_n'
  369 |     RMT.chnconf0[channel].conf_update_n = 1;
      |                           ^~~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:371:27: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'mem_rd_rst_n'
  371 |     RMT.chnconf0[channel].mem_rd_rst_n = 1;
      |                           ^~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:372:27: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'mem_rd_rst_n'
  372 |     RMT.chnconf0[channel].mem_rd_rst_n = 0;
      |                           ^~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:373:27: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'apb_mem_rst_n'
  373 |     RMT.chnconf0[channel].apb_mem_rst_n = 1;
      |                           ^~~~~~~~~~~~~
c:\Users\sukes\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:374:27: error: 'volatile union rmt_chnconf0_reg_t' has no member named 'apb_mem_rst_n'
  374 |     RMT.chnconf0[channel].apb_mem_rst_n = 0;
      |                           ^~~~~~~~~~~~~

Using library FastLED at version 3.6.0 in folder: C:\Users\sukes\Documents\Arduino\libraries\FastLED 
exit status 1

Compilation error: exit status 1

```

Pardon if I should have started a new issue, but I think my issue is related. I'm trying to run the Blink example on a [Seeed Studio XIAO ESP32C3](https://www.seeedstudio.com/Seeed-XIAO-ESP32C3-p-5431.html) and I get similar errors
```
c:\Users\jeff\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp: In member function 'void ESP32RMTController::startOnChannel(int)':
c:\Users\jeff\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:266:29: error: 'RMTMEM' was not declared in this scope
  266 |         mRMT_mem_start = & (RMTMEM.chan[mRMT_channel].data32[0].val);
      |                             ^~~~~~
c:\Users\jeff\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp: In static member function 'static void ESP32RMTController::doneOnChannel(rmt_channel_t, void*)':
c:\Users\jeff\Documents\Arduino\libraries\FastLED\src\platforms\esp\32\clockless_rmt_esp32.cpp:347:5: error: 'gpio_matrix_out' was not declared in this scope; did you mean 'gpio_iomux_out'?
  347 |     gpio_matrix_out(pController->mPin, 0x100, 0, 0);
      |     ^~~~~~~~~~~~~~~
      |     gpio_iomux_out
```
I have FastLED 3.6.0. I just updated my esp32 board definitions to 3.0.0-alpha2 and have selected 'XIAO_ESP32C3' as my board, per the instructions at https://wiki.seeedstudio.com/XIAO_ESP32C3_Getting_Started/

--- a/src/noise.cpp
+++ b/src/noise.cpp
@@ -623,7 +623,7 @@ void fill_raw_noise16into8(uint8_t *pData, uint8_t num_points, uint8_t octaves, 
 /// @param scaley the scale (distance) between y points when filling in noise
 /// @param time the time position for the noise field
 /// @todo Why isn't this declared in the header (noise.h)?
-void fill_raw_2dnoise8(uint8_t *pData, int width, int height, uint8_t octaves, q44 freq44, fract8 amplitude, int skip, uint16_t x, int scalex, uint16_t y, int scaley, uint16_t time) {
+void fill_raw_2dnoise8(uint8_t *pData, int width, int height, uint8_t octaves, q44 freq44, fract8 amplitude, int skip, uint16_t x, int16_t scalex, uint16_t y, int16_t scaley, uint16_t time) {
   if(octaves > 1) {
     fill_raw_2dnoise8(pData, width, height, octaves-1, freq44, amplitude, skip+1, x*freq44, freq44 * scalex, y*freq44, freq44 * scaley, time);
   } else {
@@ -661,7 +661,7 @@ void fill_raw_2dnoise8(uint8_t *pData, int width, int height, uint8_t octaves, u
   fill_raw_2dnoise8(pData, width, height, octaves, q44(2,0), 128, 1, x, scalex, y, scaley, time);
 }
 
-void fill_raw_2dnoise16(uint16_t *pData, int width, int height, uint8_t octaves, q88 freq88, fract16 amplitude, int skip, uint32_t x, int scalex, uint32_t y, int scaley, uint32_t time) {
+void fill_raw_2dnoise16(uint16_t *pData, int width, int height, uint8_t octaves, q88 freq88, fract16 amplitude, int skip, uint32_t x, int32_t scalex, uint32_t y, int32_t scaley, uint32_t time) {
   if(octaves > 1) {
     fill_raw_2dnoise16(pData, width, height, octaves-1, freq88, amplitude, skip, x *freq88 , scalex *freq88, y * freq88, scaley * freq88, time);
   } else {
@@ -699,7 +699,7 @@ int32_t nmin=11111110;
 /// @todo Remove?
 int32_t nmax=0;
 
-void fill_raw_2dnoise16into8(uint8_t *pData, int width, int height, uint8_t octaves, q44 freq44, fract8 amplitude, int skip, uint32_t x, int scalex, uint32_t y, int scaley, uint32_t time) {
+void fill_raw_2dnoise16into8(uint8_t *pData, int width, int height, uint8_t octaves, q44 freq44, fract8 amplitude, int skip, uint32_t x, int32_t scalex, uint32_t y, int32_t scaley, uint32_t time) {
   if(octaves > 1) {
     fill_raw_2dnoise16into8(pData, width, height, octaves-1, freq44, amplitude, skip+1, x*freq44, scalex *freq44, y*freq44, scaley * freq44, time);
   } else {

--- a/src/noise.h
+++ b/src/noise.h
@@ -134,25 +134,25 @@ void fill_raw_noise16into8(uint8_t *pData, uint8_t num_points, uint8_t octaves, 
 /// @param y y-axis coordinate on noise map (2D)
 /// @param scaley the scale (distance) between y points when filling in noise
 /// @param time the time position for the noise field
-void fill_raw_2dnoise8(uint8_t *pData, int width, int height, uint8_t octaves, uint16_t x, int scalex, uint16_t y, int scaley, uint16_t time);
+void fill_raw_2dnoise8(uint8_t *pData, int width, int height, uint8_t octaves, uint16_t x, int16_t scalex, uint16_t y, int16_t scaley, uint16_t time);
 
 /// Fill a 2D 8-bit buffer with noise, using inoise16() 
 /// @copydetails fill_raw_2dnoise8(uint8_t*, int, int, uint8_t, uint16_t, int, uint16_t, int, uint16_t)
-void fill_raw_2dnoise16into8(uint8_t *pData, int width, int height, uint8_t octaves, uint32_t x, int scalex, uint32_t y, int scaley, uint32_t time);
+void fill_raw_2dnoise16into8(uint8_t *pData, int width, int height, uint8_t octaves, uint32_t x, int32_t scalex, uint32_t y, int32_t scaley, uint32_t time);
 
 /// Fill a 2D 16-bit buffer with noise, using inoise16() 
 /// @copydetails fill_raw_2dnoise8(uint8_t*, int, int, uint8_t, uint16_t, int, uint16_t, int, uint16_t)
 /// @param freq88 starting octave frequency
 /// @param amplitude noise amplitude
 /// @param skip how many noise maps to skip over, incremented recursively per octave
-void fill_raw_2dnoise16(uint16_t *pData, int width, int height, uint8_t octaves, q88 freq88, fract16 amplitude, int skip, uint32_t x, int scalex, uint32_t y, int scaley, uint32_t time);
+void fill_raw_2dnoise16(uint16_t *pData, int width, int height, uint8_t octaves, q88 freq88, fract16 amplitude, int skip, uint32_t x, int32_t scalex, uint32_t y, int32_t scaley, uint32_t time);
 
 /// Fill a 2D 8-bit buffer with noise, using inoise16() 
 /// @copydetails fill_raw_2dnoise8(uint8_t*, int, int, uint8_t, uint16_t, int, uint16_t, int, uint16_t)
 /// @param freq44 starting octave frequency
 /// @param amplitude noise amplitude
 /// @param skip how many noise maps to skip over, incremented recursively per octave
-void fill_raw_2dnoise16into8(uint8_t *pData, int width, int height, uint8_t octaves, q44 freq44, fract8 amplitude, int skip, uint32_t x, int scalex, uint32_t y, int scaley, uint32_t time);
+void fill_raw_2dnoise16into8(uint8_t *pData, int width, int height, uint8_t octaves, q44 freq44, fract8 amplitude, int skip, uint32_t x, int32_t scalex, uint32_t y, int32_t scaley, uint32_t time);
 
 /// @} Raw Fill Functions
 

